vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/search/AbstractElementSearchAction.java

```diff
@@ -51,8 +51,15 @@
     @Inject private IWaitActions waitActions;
     @Inject private IExpectedConditions<By> expectedConditions;
     private Duration waitForElementTimeout;
+    private boolean retrySearchIfStale;
 
     public List<WebElement> findElements(SearchContext searchContext, By locator, SearchParameters parameters)
+    {
+        return findElements(searchContext, locator, parameters, false);
+    }
+
+    private List<WebElement> findElements(SearchContext searchContext, By locator, SearchParameters parameters,
+            boolean retry)
     {
         if (searchContext != null)
```

Comment:
this check can be moved to the previous method to avoid duplicate null-check in case of retry